avtozip/store/templates/store/index.html

```diff
@@ -2,7 +2,41 @@
 
 {% block title %}AvtoZip Store{% endblock %}
 
+{% block css %}
+  <link rel="stylesheet" type="text/css" href="{% static 'store/css/main.css' %}" />
+{% endblock %}
+
 {% block content %}
+  {{ block.super }}
```

Comment:
This super should not be called because we want to __substitute__ content block completely